x/pairing/keeper/qos_excellence.go

```diff
@@ -8,8 +8,7 @@ import (
 	pairingtypes "github.com/lavanet/lava/x/pairing/types"
 )
 
-func (k Keeper) UpdateProviderQos(epochPayments pairingtypes.EpochPayments) {
-}
+// TODO: implement UpdateProviderQos(payments)
```

Comment:
why?

Reply:
the epochPayments object does not exist anymore. I didn't know how the provider QoS update should've worked so I didn't know what type to put in the argument. So I replaced it with a comment